app/src/pages/inside/testCaseLibraryPage/commonMessages.ts

```diff
@@ -37,4 +37,28 @@ export const commonMessages = defineMessages({
     id: 'TestCaseLibraryPage.folders',
     defaultMessage: 'Folders',
   },
+  name: {
+    id: 'TestCaseLibraryPage.name',
+    defaultMessage: 'Name',
+  },
+  enterFolderName: {
+    id: 'TestCaseLibraryPage.enterFolderName',
+    defaultMessage: 'Enter folder name',
+  },
+  createAsSubfolder: {
+    id: 'TestCaseLibraryPage.createAsSubfolder',
+    defaultMessage: 'Create as subfolder',
+  },
+  parentFolder: {
+    id: 'TestCaseLibraryPage.parentFolder',
+    defaultMessage: 'Parent folder',
+  },
+  searchFolderToSelect: {
+    id: 'TestCaseLibraryPage.searchFolderToSelect',
+    defaultMessage: 'Search folder to select',
+  },
```

Comment:
This file is for common translations that should be used not in one place in test case library page. Some of this translation are only relevant for `createFolderModal`. You can create them in the component file itself